shell/platform/android/io/flutter/plugin/editing/InputConnectionAdaptor.java

```diff
@@ -30,6 +33,9 @@
   private InputMethodManager mImm;
   private final Layout mLayout;
 
+  // Used to determine if Samsung-specific hacks should be applied.
+  private final boolean mIsSamsung;
```

Comment:
nit: We actually don't want variables prefixed with `m`, it goes against Google Java style. The other members in the class already have it but I'd avoid matching it going forward. Ideally somebody should come back here later and clean the other names up so they fit the style guide.